src/packages/server/auth/sso/oauth2-user-profile-callback.ts

```diff
@@ -10,21 +10,21 @@
 const safeJsonStringify = require("safe-json-stringify");
 import { parseOpenIdProfile } from "@cocalc/server/auth/sso/openid-parser";
 import { UserProfileCallbackOpts } from "./types";
-
+import jwt_decode from "jwt-decode";
 export function addUserProfileCallback(opts: UserProfileCallbackOpts) {
   const { strategy_instance, userinfoURL, L2, type } = opts;
   if (userinfoURL == null) throw new Error(`the userinfoURL is required`);
-  strategy_instance.userProfile = function userProfile(accessToken, done) {
-    L2(`userinfoURL=${userinfoURL}, accessToken=${accessToken}`);
+  strategy_instance.userProfile = function userProfile(accessToken, tokenSecret, params, done) {
+    L2(`userinfoURL=${userinfoURL}, accessToken=${accessToken}, tokenSecret=${tokenSecret} `); 
```

Comment:
I'm concerned this just introduced a potential security vulnerability here.  Before the code here (and below that is now commented out) took some care to not print a secret in the logs, but now it does seem to print a secret in the logs.   Generally, we always try to avoid writing secrets to our logfiles, to reduce our attack surface.